src/runtime/step_runner.js

```diff
@@ -8,22 +8,30 @@ const {beginTiming, endTiming} = Time
 
 async function run({attachmentManager, defaultTimeout, scenarioResult, step, stepDefinition, parameterRegistry, world}) {
   beginTiming()
-  const parameters = stepDefinition.getInvocationParameters({scenarioResult, step, parameterRegistry})
-  const timeoutInMilliseconds = stepDefinition.options.timeout || defaultTimeout
-
-  let error, result
-  const validCodeLengths = stepDefinition.getValidCodeLengths(parameters)
-  if (_.includes(validCodeLengths, stepDefinition.code.length)) {
-    const data = await UserCodeRunner.run({
-      argsArray: parameters,
-      fn: stepDefinition.code,
-      thisArg: world,
-      timeoutInMilliseconds
-    })
-    error = data.error
-    result = data.result
-  } else {
-    error = stepDefinition.getInvalidCodeLengthMessage(parameters)
+  let error, result, parameters
+
+  try {
+    parameters = stepDefinition.getInvocationParameters({scenarioResult, step, parameterRegistry})
+  } catch(err) {
+    error = err
+  }
+
+  if(!error) {
```

Comment:
We've all got our pet peeves, I certainly have mine. That doesn't mean we need to share them with everyone. If minor formatting issues like this bugs you I suggest you tweak the linting rules, or write your own if you can't find rules that are strict enough for your liking. 

Or just cut people some white space slack and fix it yourself after you merge.